lib/views/track/track_view.dart

```diff
@@ -102,9 +102,12 @@ class _TrackViewState extends ConsumerState<TrackView>
     return tracks.when(
       skipLoadingOnRefresh: false,
       data: (data) => _buildScaffoldWithData(context, data, ref),
-      error: (err, stack) => MeditoErrorWidget(
-        message: err.toString(),
-        onTap: () => ref.refresh(tracksProvider(trackId: widget.id)),
+      error: (err, stack) => SizedBox(
+        height: MediaQuery.of(context).size.height / 2,
```

Comment:
Would it be better to wrap the content rather than take up half the screen? Not sure what is happening here so just wondering

Reply:
The MeditoErrorWidget is a separate widget wrapped with a Scaffold, taking the complete screen height. So, in the bottom sheet, it's opening the bottom sheet on full screen. So, I thought in the error case it would be good if we can show the error in half screen because it's a bottom sheet. Lmk, if you have any suggestion ;)

Reply:
For now I added a check for scaffold lmk if it's looking good to you.